Cargo.toml

```diff
@@ -40,6 +40,7 @@ dirs = "4.0.0"
 emoji = "0.2.1"
 gh-emoji = "1.0.7"
 hostname = "0.3.1"
+image = "0.24.6"
```

Comment:
Could you please disable the default features of the image crate and enable only png? This should reduce the amount of dependencies it is pulling in.